core/src/main/scala/com/karumi/shot/Shot.scala

```diff
@@ -84,7 +84,8 @@ class Shot(adb: Adb,
                         buildType: String,
                         projectName: String,
                         shouldPrintBase64Error: Boolean,
-                        tolerance: Double): ScreenshotsComparisionResult = {
+                        tolerance: Double,
+                        onlyFailing: Boolean): ScreenshotsComparisionResult = {
```

Comment:
Can we be more explicit with the variable name? Something like ``showOnlyFailingTestsInReports`` or something similar?

Reply:
Sure, I'll change it